docs/troubleshooting-guide/models.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Troubleshooting models
+---
+
+# Troubleshooting models
+
+What kind of problem are you having with your [model][model-docs]?
+
+- [Cannot create a model](#cannot-create-a-model).
+- [Cannot edit or save changes to a model](#cannot-edit-or-save-changes-to-a-model).
+- [Model performance is poor](#model-performance-is-poor).
+- [Model doesn't work with data sandboxing][troubleshooting-sandboxing].
+
+## Cannot create a model
+
+If you don't see [the model button][model-button-image] (three squares):
+
+1. Check if you're using a Metabase version that's 0.42.0 or greater from the **gear icon** > **About Metabase**.
+2. Clear your browser cache.
+3. Ask your Metabase admin to clear the proxy cache (if you're using one).
+4. Ask your Metabase admin if [nested queries are enabled][nested-query-settings-docs] under **Admin** > **Settings** > **General**.
+
+## Cannot edit or save changes to a model
+
+If your changes to a model's metadata or underlying question aren't showing up:
+
+1. Refresh your browser to confirm you're not viewing cached results.
+2. Search for [known bugs or limitations][known-issues] using the label [`Querying/Models`][known-issues-models].
+
+## Model performance is poor.
+
+1. Optimize the underlying saved question or SQL query.
+  - [Ask for less data][limit-data-learn].
+  - For models that use SQL questions, [look for bottlenecks using SQL EXPLAIN][sql-explain-learn].
+2. Optimize your database schemas.
+  - [Aggregate data ahead of time with summary tables][summary-tables-learn].
+  - [Index frequently queried columns][indexes-learn].
+  - [Denormalize your data][denormalize-data-learn].
+  - [Materialize views][materialize-views-learn].
+  - [Pull data out of JSON and slot its keys into columns][flatten-json-learn].
+3. Optimize your data warehouse(s) or database(s).
+  - [Replicate your database][replicate-database-learn].
+  - [Consider a database optimized for analytics][analytics-database-learn].
+
+**Explanation**
+
+Models are a type of saved question, so they will only perform as fast as the original question or SQL query.
+
+If you want to improve the performance of a model, you can make optimizations at the query, schema, or database level (depending on your data permissions, technical expertise, and willingness to tinker).
+
+## Are you still stuck?
+
+If you can’t solve your problem using the troubleshooting guides:
+
+- Search or ask the [Metabase community][discourse].
+- Search for [known bugs or limitations][known-issues] using the label [`Querying/Models`][known-issues-models].
```

Comment:
Can we drop this first link, or at least make it the second link in this sentence? I would expect to click on something here and be taken to a list of current issues with models (which is exactly what the second link does). I'd also drop the code case for the `Querying/Models` and just make it a normal text link.